src/pymorize/units.py

```diff
@@ -34,6 +38,23 @@
 ureg = pint_xarray.unit_registry
 
 
+# Add PSU to the unit registry for salinity conversions:
+# ureg.define("PSU = 1 * ppt")
+# ureg.define("0.001 = ppt")
+
+
+@xr.register_dataarray_accessor("pint")
+class PSUAwarePintDataArrayAccessor(pint_xarray.accessors.PintDataArrayAccessor):
+    def to(self, unit):
+        if unit == "0.001":
+            logger.critical(">>>> PG PART!!!!")
+            return super().to("ppt")
+        return super().to(unit)
```

Comment:
@siligam: this is what I had in mind, but you would still need to fix it. Unfortunately, `ppt` (should be parts per thousand) is already defined as `picopint`.

I don't think we should override a built-in `pint` unit at this point; rather just handle the conversion logic manually. This conversion override is where you should probably include the corrected code.

Reply:
This is what this looks like in practice:

```python
>>> import xarray as xr
>>> da = xr.DataArray([10.0], attrs={"units": "psu"})  # Comes out of FESOM
>>> # The above is equivalent to:
>>> ds = xr.open_dataset("/my/fesom/file.nc")  # Attribute sst is the variable name
>>> da = ds.sst
>>> da.pint.quantify() # Turns it into a "unit-aware" array
>>> target_unit = "0.001"
>>> new_da = da.pint.to(target_unit)
```